standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnStore.java

```diff
@@ -132,6 +132,7 @@ enum MUTEX_KEY {
   char MINOR_TYPE = 'i';
   char REBALANCE_TYPE = 'r';
   char ABORT_TXN_CLEANUP_TYPE = 'c';
+  char SMART_OPTIMIZE_TYPE = 's';
```

Comment:
IDK, what if we use `*`?

Reply:
We should also update the SQL mapping since we added a new type
````
CASE WHEN CC_TYPE = 'i' THEN 'minor' WHEN CC_TYPE = 'a' THEN 'major' ELSE 'UNKNOWN' END,
````

Reply:
Done